lldb/tools/lldb-dap/src-ts/extension.ts

```diff
@@ -17,14 +17,46 @@ function createDefaultLLDBDapOptions(): LLDBDapOptions {
       const path = vscode.workspace
         .getConfiguration("lldb-dap", session.workspaceFolder)
         .get<string>("executable-path");
-      if (path) {
-        return new vscode.DebugAdapterExecutable(path, []);
+
+      if (!path) {
```

Comment:
even if the path is not set, we should check if the `packageJSONExecutable` actually exists
By default, the `path` is not set. As such, new users would not get this error message, although those are the ones which are most likely to run into issues